src/Microsoft.Framework.Runtime/DependencyManagement/NuGetDependencyResolver.cs

```diff
@@ -222,11 +222,8 @@ public void Initialize(IEnumerable<LibraryDescription> packages, FrameworkName t
         {
             var defaultHashPath = defaultResolver.GetHashPath(package.Id, package.Version);
```

Comment:
I had a look at the `perfomance_stuff` branch change on this. I probably need to understand this a bit better. It looks like you still do a disk read to read the contents of the file in `for` loop during the first time. Am I missing something here?